packages/twenty-front/src/modules/ui/input/components/TextArea.tsx

```diff
@@ -57,6 +73,8 @@ export const TextArea = ({
 }: TextAreaProps) => {
   const computedMinRows = Math.min(minRows, MAX_ROWS);
 
+  const inputId = useId();
```

Comment:
we usually use `v4()`. Do we need an id that is not an `uuid` maybe?

Reply:
I documented this change in my issue about using real `<label>` HTML elements for inputs. (<https://github.com/twentyhq/twenty/issues/7281>)

I saw that a UUID v4 was generated in a render function to serve as an input id: https://github.com/twentyhq/twenty/blob/2f41e1405fb206a4826e267eb53781b5d307eac5/packages/twenty-front/src/modules/ui/input/components/Checkbox.tsx#L168

UUIDs are expensive to compute as cryptographically secure identifiers with low collision probabilities. However, these properties aren't necessary for an input's ID.

It's also better to have a consistent identifier that doesn't change every time. Assistive technologies like screen readers will behave better with a consistent identifier.

Furthermore, generating a UUID in a render function will re-compute it every time the component is rendered. As generating a UUID comes with a cost, it's better to either memoize it or not use an UUID.

The new `useId()` React hook returns a short and memoized ID with no collision probability. It also works well with server-side rendering, as the ID will be the same whether the component is rendered client-side or server-side.